astrovascpy/utils.py

```diff
@@ -257,12 +257,15 @@ def create_entry_largest_nodes(graph, params):
         if depth_ratio > 1:
             depth_ratio = 1.0
             L.warning("The depth_ratio must be <= 1. Taking depth_ratio = 1.")
-        positions = graph.points
+
+        degrees = GRAPH_HELPER.get_degrees(graph)
```

Comment:
Usage of `GRAPH_HELPER` class can lead to unexpected results. I suggest an object-oriented approach in #15 which can be merged on top of this change.